packages/terra-form/src/Textarea.scss

```diff
@@ -1,14 +1,78 @@
 :local {
   .textarea {
-    border: #dedfe0 solid 1px;
-    border-radius: 0.25em;
-    font-size: 1em;
-    line-height: 1.25;
-    padding: 0.178em;
+    background-color: var(--terra-form-textarea-background-color, #fff);
+    background-image: var(--terra-form-textarea-focus-animation-background, linear-gradient(to bottom, #396ee5, #396ee5));
+    background-position: 0 100%;
+    background-repeat: no-repeat;
+    background-size: 0 2px;
+    border-color: var(--terra-form-textarea-border-color, #dedfe0);
+    border-radius: var(--terra-form-textarea-border-radius, 0);
+    border-style: var(--terra-form-textarea-border-style, solid);
+    border-width: var(--terra-form-textarea-border-width, 1px);
+    color: var(--terra-form-textarea-color, #000);
+    cursor: var(--terra-form-textarea-cursor, auto);
+    font-size: var(--terra-form-textarea-font-size-small, 1em);
+    line-height: var(--terra-form-textarea-line-height, 1.25);
+    opacity: var(--terra-form-textarea-focus-opacity, 0.5);
+    padding: var(--terra-form-textarea-padding, 0);
+    transition: var(--terra-form-textarea-focus-transition, background-size ease 0.2s);
     width: 100%;
 
     &[required] {
       box-shadow: unset;
     }
+
+    &[disabled] {
+      background: var(--terra-form-textarea-disabled-background, rgba(0, 0, 0, 0.05));
+      border-color: var(--terra-form-textarea-disabled-border-color, #000);
+      border-style: var(--terra-form-textarea-disabled-border-style, solid);
+      border-width: var(--terra-form-textarea-disabled-border-width, 1px);
+    }
+
+    &:focus {
+      background-size: var(--terra-form-textarea-focus-mouse-animation-size, 0 0);
+      border-color: var(--terra-form-textarea-focus-mouse-border-color, #dedfe0);
+      border-style: var(--terra-form-textarea-focus-mouse-border-style, solid);
+      border-width: var(--terra-form-textarea-focus-mouse-border-width, 1px);
+      box-shadow: var(--terra-form-textarea-focus-mouse-box-shadow, 1px 1px 1px 1px #46b1eb);
+      opacity: var(--terra-form-textarea-focus-mouse-opacity, 1);
+      outline: none;
+    }
+
+    [data-whatinput='keyboard'] &:focus {
```

Comment:
When are we going to be adding input detection in? Should we have these separate styles before it's been implemented?

Reply:
We can add it in this issue. We've landed on the implementation details, and can easily add it in this issue. https://github.com/cerner/terra-core/issues/972

Reply:
I think we should.

Reply:
Agreed, we may want to pick up #972 before we start the theming work for textarea.